### ISSUE TITLE
connectome2tck: exporting exclusive edges

### ISSUE THREAD

**Issue by ppruc** [MRtrix3/mrtrix3#2705]
Hi everyone, 

when trying to export all edges from a connectome using connectome2tck, I realised that it outputs the edges in both directions, e.g. for nodes 1 + 2, edge_1-2.tck and edge 2-1.tck are exported (files are identical in their content). I was able to circumvent this behaviour by running

```
connectome2tck tracts.tck parcels_assignments.csv edge_ -nodes 1:256 -exclusive 
```

It would maybe be helpful for other users to have a commandline option that allows controlling this behaviour directly?

Cheers 
Philip

**Comment by Lestropie** [comment]
For clarification: While the `-exclusive` command-line option achieves the desired behaviour ie. each edge is generated only once, it is somewhat shoe-horned to trigger on the presence of that command-line option, which is really there for a distinctly different purpose and that is reflected in the documentation. What is suggested is that a *unique* command-line option be used that specifically controls whether, when files are generated on a per-edge basis, a duplicate of each edge should be generated (which facilitates easy filesystem searching for a set of files corresponding to a given edge) or whether each edge should generate just one file.

Note that this would have a non-negligible interaction with #2683, since with that feature implemented, the two files corresponding to each edge would in fact not be unique, since each streamline would have the vertices in reverse order between the two.
